requirements/uninstall-requirements.txt

```diff
@@ -16,3 +16,4 @@ eventlet
 contextlib2
 redis-py-cluster
 enum34
+celery # Remove once we are back to regular celery releases
```

Comment:
I think this '#' is messing with the uninstall script:
https://github.com/dimagi/commcare-hq/blob/3b0d3ba117d67ccd5798b55687e31e557060a030/scripts/uninstall-requirements.sh#L7

Reply:
fixed here 8cebcae6cce